src/Bundle/Doctrine/ORM/ExpressionBuilder.php

```diff
@@ -18,8 +18,9 @@
 use Doctrine\ORM\Query\Expr\Join;
 use Doctrine\ORM\QueryBuilder;
 use Sylius\Component\Grid\Data\ExpressionBuilderInterface;
+use Sylius\Component\Grid\Data\MemberOfAwareExpressionBuilderInterface;
 
-final class ExpressionBuilder implements ExpressionBuilderInterface
+final class ExpressionBuilder implements ExpressionBuilderInterface, MemberOfAwareExpressionBuilderInterface
```

Comment:
```suggestion
final class ExpressionBuilder implements MemberOfAwareExpressionBuilderInterface
```
imo it is not needed as the new interface extends the old one